### PR TITLE
Gui 1587 Added options to view shared bucket, download/upload from/to shared bucket.

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-1587

### CODE REVIEW

**Comment by dkavanagh** [comment]
Looks like the failures were not critical and likely random.

**Comment by aliceh** [comment]
Good to merge. One random fail in set_all_to_list view.

**Comment by aliceh** [comment]
Good to merge.

File: eucaconsole/templates/buckets/buckets.pt [hunk 1]
```diff
@@ -20,9 +20,26 @@
...
+                    </li>
+                    <li>
+                        <a id="upload-object-btn" i18n:translate="" target="_blank"
+                            href="${request.route_path('bucket_upload', name='__shared__', subpath='')}">Upload Object(s)</a>
+                    </li>
+                </ul>
             </div>
             <div metal:fill-slot="tile_header">
-                <a ng-href="{{ item.contents_url }}" class="has-bucket-icon">
+                <a ng-href="{{ item.contents_url }}" class="has-shared-icon" ng-show="{{ item.shared }}">
```
Comment (kamalgill) [inline eucaconsole/templates/buckets/buckets.pt]: @dkavanagh: Unlike ng-href, ng-show and ng-hide expect an expression as the attribute value, so we should remove the curly braces here and elsewhere.  See https://docs.angularjs.org/api/ng/directive/ngShow

Comment (dkavanagh) [inline eucaconsole/templates/buckets/buckets.pt]: good catch.

**Comment by dkavanagh** [comment]
fixed curly braces in 4 places. Good to merge?

**Pull request merged**